qa/rpc-tests/exodus_sendmint.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+from test_framework.authproxy import JSONRPCException
+from test_framework.test_framework import ExodusTestFramework
+from test_framework.util import assert_equal, assert_raises_message
+
+class ExodusSendMintTest(ExodusTestFramework):
+    def run_test(self):
+        super().run_test()
+
+        sigma_start_block = 500
+
+        self.nodes[0].generatetoaddress(100, self.addrs[0])
+        self.nodes[0].generate(sigma_start_block - self.nodes[0].getblockcount())
+
+        assert(sigma_start_block == self.nodes[0].getblockcount())
```

Comment:
Why do we use `assert` instead of `assert_equal`?

Reply:
good idea to change to `assert_equal`.